### PR TITLE
IE Fix

### PR DESCRIPTION
**Pull request by mwernimont**
Global Variable to use to have different functionality for IE

### CODE REVIEW

File: layout/js/map.js [hunk 1]
```diff
@@ -14,6 +14,20 @@ var colors = {
...
+  //IE Fix
+  var ua = window.navigator.userAgent;
+  //IE10 and Below
+  var msie = ua.indexOf("MSIE ");
+  //IE11
+  var trident = ua.indexOf('Trident/');
+  //IE Edge
+  var edge = ua.indexOf('Edge/');
+  if(msie > 0 || trident > 0 || edge > 0){
+    console.log('Internet Explorer');
```
Comment (jiwalker-usgs) [inline layout/js/map.js:26]: does console.log work in IE?

Comment (mwernimont) [reply layout/js/map.js:26]: I do not know, I had alert, but figured you guys didn't want that popping up each time

**Comment by ldecicco-USGS** [comment]
I got trigger happy. It's different now, but not fixed. I'll describe in the issue

**Comment by mwernimont** [comment]
It is the start to the fix, JR needs to work his magic first

**Pull request merged**